docker/README.md

```diff
@@ -38,8 +38,8 @@ We also provide [example compose files](examples/README.md) that integrate colle
 such as Kafka or Elasticsearch.
 
 ## Configuration
-Configuration is via environment variables, defined by [zipkin-server](https://github.com/openzipkin/zipkin/blob/master/zipkin-server/README.md). Notably, you'll want to look at the `STORAGE_TYPE` environment variables, which
-include "cassandra", "mysql" and "elasticsearch".
+Configuration is via environment variables, defined by [zipkin-server](https://github.com/openzipkin/zipkin/blob/master/zipkin-server/README.md). Notably, you'll want to look at the `ZIPKIN_STORAGE` environment variables, which
```

Comment:
While I don't question the naming change I want to make sure we support BOTH to ease migration.

Reply:
@mrproliu We could change this back. I think.

Reply:
Got it, I will make these environments both work.